src/client/components/PageChantier/Indicateurs/Indicateurs.tsx

```diff
@@ -26,41 +25,30 @@ export default function Indicateurs({ indicateurs, détailsIndicateurs, estDispo
         Indicateurs
       </Titre>
       { 
-        listeRubriquesIndicateurs.map(rubrique => (
-          <section
-            className='fr-mb-3w'
-            id={rubrique.ancre}
-            key={rubrique.ancre}
-          >
-            <Titre
-              baliseHtml='h3'
-              className='fr-text--lg fr-mb-1w'
+        indicateurs.map(indicateur => {
```

Comment:
En itérant sur `indicateurs`, les indicateurs sont trié par ordre croissant de leur id et on perd le tri sur leur typologie (info qu'on stocke dans le front dans `listeRubriquesIndicateurs`) : les indicateurs d'impact sont en premier et les indicateurs de contexte en dernier